### ISSUE TITLE
Error during Installation

### ISSUE THREAD

**Issue by sectester22** [nccgroup/autochrome#34]
Hey there,

I used autochrome for a couple of months on Kali Linux. After the last systemupdate and rebuilding autochrome for the new Chromium Version I get the following Error message:

```
└─$ ruby autochrome.rb -c
[???] Detected Linux (x86_64)
[---] Determining latest Chromium version for Linux_x64
[---] Got latest version: Linux_x64 1035024
[---] Downloading Chromium from https://commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/1035024/chrome-linux.zip
[???] Saved  160036982 of 160036982 bytes (100%)
[---] Installed Chromium
/home/user/autochrome/lib/auto_chrome/prefs.rb:87:in `initialize': EVP_PKEY_new_mac_key: malloc failure (OpenSSL::HMACError)
	from /home/user/autochrome/lib/auto_chrome/prefs.rb:87:in `new'
	from /home/user/autochrome/lib/auto_chrome/prefs.rb:87:in `sign'
	from /home/user/autochrome/lib/auto_chrome/prefs.rb:69:in `[]='
	from /home/user/autochrome/lib/auto_chrome/profile.rb:79:in `block in init_prefs'
	from /home/user/autochrome/lib/auto_chrome/profile.rb:79:in `each'
	from /home/user/autochrome/lib/auto_chrome/profile.rb:79:in `init_prefs'
	from /home/user/autochrome/lib/auto_chrome/profile.rb:27:in `initialize'
	from /home/user/autochrome/lib/auto_chrome/profile_builder.rb:27:in `new'
	from /home/user/autochrome/lib/auto_chrome/profile_builder.rb:27:in `block in generate'
	from /home/user/autochrome/lib/auto_chrome/profile_builder.rb:26:in `map'
	from /home/user/autochrome/lib/auto_chrome/profile_builder.rb:26:in `generate'
	from /home/user/autochrome/lib/auto_chrome.rb:61:in `go'
	from autochrome.rb:91:in `<main>'
```

While on a fresh Debian Installation it works like a charm, it still throws that Error on a fresh Kali Linux Install.
Is there a possible Workaround? 

Thanks in Advance!

**Comment by sectester22** [comment]
The installed openssl Version:
Package: openssl
Version: 3.0.4-2

**Comment by sectester22** [comment]
Seems openssl Version 1.1.1n-0  works, while openssl Version: 3.0.4-2 isnt anymore

**Comment by dwalter0** [comment]
I just made a pull request that fixed this issue for me.

**Comment by peter-h-ncc** [comment]
Fixed #39
